src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -5621,13 +5621,32 @@ namespace winrt::TerminalApp::implementation
             if (const auto activeControl = _GetActiveControl())
             {
                 const auto profileName = activeControl.Settings().ProfileName();
-                const std::wstring fullCommandline = activeControl.Settings().Commandline().c_str();
+                std::wstring fullCommandline = activeControl.Settings().Commandline().c_str();
+
+                // If the commandline starts with a quote ("), we just need the string up until the
+                // next quote, and we need that string to be stripped of whitespace
```

Comment:
I'm not quite sure I understand why we remove the spaces. Wouldn't that turn
```
"C:\Program Files\Microsoft Visual Studio\2022\Preview\Common7\Tools\VsDevCmd.bat"
```
into
```
C:\ProgramFiles\MicrosoftVisualStudio\2022\Preview\Common7\Tools\VsDevCmd.bat
```
?

Reply:
I was hung up on conforming to the way we do it in `MangleStartingDirectoryForWSL`, updated it to make more sense now